src/pages/homeCollection/kitAssembly.js

```diff
@@ -215,16 +220,16 @@ const editAssembledKits = () => {
         document.getElementById('scannedBarcode').value = editKitObj[conceptIds.returnKitTrackingNum]
         document.getElementById('supplyKitId').value = editKitObj[conceptIds.supplyKitId]
         document.getElementById('returnKitId').value = editKitObj[conceptIds.returnKitId]
-        document.getElementById('cupId').value = editKitObj[conceptIds.collectionCupId]
-        document.getElementById('cardId').value = editKitObj[conceptIds.collectionCardId]
+        document.getElementById('cupId').value = editKitObj[conceptIds.collectionCupId].slice(0, -3) + " " + editKitObj[conceptIds.collectionCupId].slice(-3)
+        document.getElementById('cardId').value = editKitObj[conceptIds.collectionCardId].slice(0, -3) + " " + editKitObj[conceptIds.collectionCardId].slice(-3)
         appState.setState({UKID: editKitObj[conceptIds.UKID]})
       });
     }); // state to indicate if its an edit & also pass the UKID
 }}
 
-const checkCollecitonUniqueness = async (collectionUniqueId) => {
+const checkUniqueness = async (supplyKitId, collectionId) => {
   const idToken = await getIdToken();
-  const response = await fetch(`${baseAPI}api=collectionUniqueness&id=${collectionUniqueId}`, {
+  const response = await fetch(`${baseAPI}api=collectionUniqueness&supply=${supplyKitId}&collection=${collectionId}`, {
```

Comment:
For clarity, should we use `collectionId` instead of 'collection'  and `supplyKitId` instead of 'supply' in `supply=${supplyKitId}&collection=${collectionId}`?